buildchain/buildchain/salt_tree.py

```diff
@@ -362,16 +362,17 @@ def _get_parts(self) -> Iterator[str]:
         "salt/metalk8s/addons/prometheus-operator/deployed/",
         "service-configuration.sls",
     ),
-    Path("salt/metalk8s/addons/ui/deployed/dependencies.sls"),
-    Path("salt/metalk8s/addons/ui/deployed/ingress.sls"),
-    Path("salt/metalk8s/addons/ui/deployed/init.sls"),
-    Path("salt/metalk8s/addons/ui/deployed/files/metalk8s-ui-deployment.yaml"),
-    Path("salt/metalk8s/addons/ui/deployed/namespace.sls"),
+    Path("salt/metalk8s/addons/shell-ui/deployed/ingress.sls"),
+    Path("salt/metalk8s/addons/shell-ui/deployed/init.sls"),
+    Path("salt/metalk8s/addons/shell-ui/deployed/files/shell-ui-deployment.yaml"),
+    Path("salt/metalk8s/addons/shell-ui/deployed/namespace.sls"),
```

Comment:
Same story here, I don't think the Shell deserves its own set of Salt formulas. IMO, we should treat both "UIs" as a single deployable unit (so, a single Ingress, a single namespace, etc.).
Splitting things would only make sense outside of MetalK8s, which is not really a target at the moment.